website/source/docs/connect/proxies.html.md

```diff
@@ -16,8 +16,8 @@ can chose other proxies to plug in as well. This section describes how to
 configure Envoy or the built-in proxy using Connect, and how to integrate the
 proxy of your choice.
 
-To ensure that services only meke allow external connections are established via
-the Connect protocol they should only accept connections on a loopback address.
+To ensure that services only allow external connections established via
+the Connect protocol, they should only accept connections on a loopback address.
```

Comment:
```suggestion
the Connect protocol, you should configure Connect to only accept connections on a loopback address.
```
This seems a little more clear on the operators responsibilities.

Reply:
```suggestion
the Connect protocol, you should configure all services to only accept connections on a loopback address.
```